009_team_7_agent_design/009_team_7_agent_design.tex

```diff
@@ -1 +1,443 @@
+    \item  \texttt{prevAge}: The age of the agent on the previous day.
+    \item  \texttt{prevHP}: A record of the HP at the end of the previous day.
+    \item  \texttt{foodEaten}: The amount of food consumed the previous day.
+    \item  \texttt{receivedReq}: This is a variable that indicates whether the agent has decided to abide by a request made in a message. A false value indicates that no requests have been accepted. 
+    \item  \texttt{takeFood}: This stores the amount of food an agent has requested to be taken. It is initialised with a value of -1 if there are no accepted requests.
+    \item  \texttt{leaveFood}: This stores the amount of food an agent has requested to be left. It is also initialised with a value of -1 if there are no accepted requests.
+\end{itemize}
+
+\subsection{Floor Risk Estimation}
+\label{subsec: Floor Map}
+Upon a floor reshuffle, the agent uses its experiences from past floors in order to build an expectation of the level of risk it associates with the new floor that it has been assigned to. The first step to doing this is to generate a prediction for the amount of food that will be available on the current floor. 
+
+In order to make this estimate, the agent utilises the prevFloors map from its memory. This contains information on the average food that was available on each floor that the agent has already been on. The agent approximates the relationship between the floor level and the average food available on that floor. For example, let us take an agent that has been on floors 10 and 6, and is now assigned to floor 8. The agent will calculate the linear function which corresponds to the average food on floors 6 to 10. It will then use this function to obtain the estimate for the food it expects on floor 8. 
+
+The above assumes that the agent has experienced floors both above and below the current floor. In the scenario where the agent has only experienced floors either below or above its current floor, the agent will extrapolate the trend line for the average food from the closest experienced floor. 
+
+Next, the agent determines the level of risk to its survival that it associates with the current floor. It does this by checking whether the estimated amount of food would be enough to exit a hypothetical critical state and enter the weak state. The difference between the required food and the estimated food will determine the floor risk value. The floor risk value will subsequently adjust the agent's greediness. 
+
+Note that since higher levels of strategising are associated more with conscientiousness agents, it makes sense for their estimates to be more pessimistic. Therefore, they will under-estimate food availability and the floor risk will have a greater impact on their greediness. In the case of less conscientiousness agents, who would be both less planning-oriented and also more optimistic, the food estimation is higher and the risk estimator is weighed down.
```

Comment:
```suggestion
Note that since higher levels of strategising are associated more with conscientiousness agents, their estimates will be more pessimistic. Therefore, they will underestimate food availability and the floor risk will have a greater impact on their greediness. In the case of less conscientiousness agents, who would be both less planning-oriented and also more optimistic, the food estimation is higher and the risk estimator is weighed down.
```